src/EditorFeatures/CSharpTest/Completion/CompletionProviders/OverrideCompletionProviderTests.cs

```diff
@@ -2549,8 +2840,8 @@ public override void goo()
     {
         base.goo();$$
     }
-#if true
-#endif
+    #if true
```

Comment:
not sure if this test is showing desired behavior, or just showing what it happened to do at the time.  To me, the new version seems better as it leaves the original indentation unchanged.